back/engines/commercial/admin_api/app/jobs/admin_api/bulk_delete_users_job.rb

```diff
@@ -1,11 +1,10 @@
 class AdminApi::BulkDeleteUsersJob < ApplicationJob
   def run(emails)
     emails.each do |email|
-      user = User.find_by_cimail(email) || raise("No user with email #{email} found")
-      DeleteUserJob.perform_later(user)
-    rescue StandardError => e
-      Sentry.capture_exception(e)
-      Rails.logger.error(([e.message] + e.backtrace).join("\n"))
+      ErrorReporter.handle do
+        user = User.find_by_cimail(email) || raise("No user with email #{email} found")
```

Comment:
I've seen this pattern multiple times in the code base already where we cell `User.find_by_cimail` and want to raise if it does not exist.

I think we could introduce `User.find_by_cimail!` that throws a `ActiveRecord::RecordNotFound` error which would makes lives a bit easier. Not in scope of this PR, just something I have noticed.